Persephone/Views/AlbumItemView.swift

```diff
@@ -59,6 +59,10 @@ class AlbumItemView: NSView {
     hidePlayButton()
   }
 
+  @objc func viewDidScroll(_ notification: Notification) {
+    viewWillScroll(notification)
```

Comment:
Could calling `viewWillScroll` _from_ `viewDidScroll` be a little confusing? Could both methods call a shared implementation?

Reply:
Yeah, I could just call `hidePlayButton()` in each.